Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -320,11 +326,11 @@ public void onAdStart(@NonNull String placementReferenceId) {
             if (mPlacementId.equals(placementReferenceId)) {
                 mIsPlaying = true;
                 //Let's load it again to mimic auto-cache
-                if (AdSize.isBannerAdSize(mAdConfig.getAdSize())) {
-                    sVungleRouter.loadBannerAd(mPlacementId, mAdConfig.getAdSize(), mVungleRouterListener);
-                } else if (VUNGLE_MREC == mAdConfig.getAdSize()) {
-                    sVungleRouter.loadAdForPlacement(mPlacementId, mVungleRouterListener);
-                }
+//TODO                if (AdSize.isBannerAdSize(mAdConfig.getAdSize())) {
```

Comment:
Hi @palrahul @alexander-gubov 
I think we should think about this part: how to auto cache banner/MREC if previous ads bided with adMarkup. I am not sure if it's ok to keep the origin logic we did on the left side.

Reply:
There is no way, But you can keep the same logic it will just fail since you don't pass admarkup.
Some logs suppressions maybe needed

Reply:
should be fine, however I'm not sure why we even pass mVungleRouterListener. Even if it fails we should not have generated the error